htdocs/kernel/menusitems.php

```
@@ -0,0 +1,263 @@
+    /**
+     * @return mixed
+     */
+    public function get_new_enreg()
+    {
+        global $xoopsDB;
+        $new_enreg = $xoopsDB->getInsertId();
+
+        return $new_enreg;
+    }
+
+    public function getFormItems($category_id, $action = false)
```

PHPMD (2.15.0)

[error] The method getFormItems has a boolean flag argument $action, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)